### DISCUSSION TITLE
请求更新docker哇

### DISCUSSION THREAD

**Discussion by RayXu14** [Mrs4s/go-cqhttp#869] [Ideas]
0.9.40-fix5-ffmpeg，我看了pcrbot/gocqhttp:0.9.40-fix4-ffmpeg的image layers感觉抄不来dockerfile，有些地方是长串数字，而且没看到版本号在哪里设定

**Comment by Ink-33** [comment]
啥...啊，理解不了你说的啥

**Comment by RayXu14** [comment]
啊……大佬们对不起，我看错人了_(:зゝ∠)_我自己想办法去
